src/elements/common/messages.js

```diff
@@ -317,6 +317,26 @@ const messages = defineMessages({
         description: 'Error text for rename dialog when name is too long',
         defaultMessage: 'This name is too long.',
     },
+    fileRenameHeading: {
```

Comment:
to keep with the pattern of the other ids, maybe `renameDialogFileHeader`?

Reply:
that is my initial thought. however the itemType are all set as all lowercase such as `file` and `folder`. I was think about two approaches
1. name message id like above and wrote `<FormattedMessage {...messages[`${itemType}RenameHeading`]} />`
2. name message id like `renameDialogFileHeading` and wrote `<FormattedMessage {...(type === 'file' ? messages.renameDialogFileHeading : messages.renameDialogFolderHeading)} />`
3. update first letter in `type` to upperCase

I didn't like the second approach because it doesn't give good clarity and readability. Also it lacks extensibility if we want to support more itemTypes in contentExplorer. Nor third approach either because I feel like it is extra implantation just for making messages id aligned. Let me know what you think or any suggestions. @tjuanitas

Reply:
it's probably just more readable and cleaner to use a switch statement or a map:

```
const headerMessages = {
    [ITEM_TYPE_FILE]: ...,
    [ITEM_TYPE_FOLDER]: ...,
    [ITEM_TYPE_WEBLINK]: ...,
};

<Modal.Header>
  {formatMessage(headerMessages[type])}
</Modal.Header>
```
or
```
let header;

switch (type) {
  case "file":
    header = ...
    break;
  case "folder":
    header = ...
    break;
  case "web_link":
    header = ...
    break;
  default:
    break;
}
```